### PR TITLE
Fix/phase1

### PR DESCRIPTION
**Pull request by ppeigne**

### CODE REVIEW

File: day01/resources/are_blue_pills_magics_description.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Patient: number of the patient.
+
+#Micrograms: quantity of blue pills patient has takern (in micrograms).
```
Comment (madvid) [inline day01/resources/are_blue_pills_magics_description.txt]: patient has taken

File: day01/resources/are_blue_pills_magics_description.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+#Score: Standardized score at the spacecraft drivimng test.
```
Comment (madvid) [inline day01/resources/are_blue_pills_magics_description.txt]: ... the spacecraft driving test

**Comment by Talasta** [comment]
Waiting for fixs then I will approve.

File: day03/ex09/ex09.md [hunk 3]
```diff
@@ -2,14 +2,16 @@
...
 |                         |                         |
 | -----------------------:| ----------------------- |
-|   Turning directory :   |  ex09                   |
+|   Turn-in directory :   |  ex09                   |
 |   Files to turn in :    |  log_reg.py             |
-|   Forbidden function :  |  None                   |
+|   Forbidden functions : |  None                   |
 |   Remarks :             |  n/a                    |
+|   Ressources :          |   	Machine Learning Mooc / Week 3 - Regularized Logistic Regression: https://www.coursera.org/learn/machine-learning/lecture/4BHEy/regularized-logistic-regression | 
+
```
Comment (fxbabin) [inline day03/ex09/ex09.md]: resource/links format differs from previous exercices

File: day02/resources/resource_links.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Link to train_dataset_clean.csv and test_dataset_clean.csv:
+https://drive.google.com/file/d/1thPCFmGDTmD8QDuvvlrvhUxXtRiqrcX_/view
```
Comment (fxbabin) [inline day02/resources/resource_links.txt]: maybe create a markdown link or a bitly url

File: day03/ex09/ex09.md (hunk 3 above)
Comment (matboivin) [reply day03/ex09/ex09.md]: Hi! It's a good remark! Maybe `resources` would be more appropriate.

Comment (madvid) [reply day03/ex09/ex09.md]: Helpful links sounds better for me.
One should understand that the video is not compulsory and this is just a tiny help to dig in the right direction if it is needed.
folks who are working on the bootcamp must not understand that it is compulsory and the only ressource they should look at.

**Comment by madvid** [comment]
@blopax tu peux faire ta PR en te concentrant sur le day04.
Surtout les headers des exercices pour spécifier les helpful links et les docstrings.

**Comment by madvid** [comment]
News commit but it is still for harmonization,

I just modify all the title sections of all the exercises of all the days which were given this way:
'**[title section]**'
and i modify in the following style:
'## [title section]' or '### [title section]'

I also modify slightly the exercise 08 of the day03 on the ridge method into 2 parts as it was a big exercise.

**Pull request merged**